configure.ac

```diff
@@ -29,7 +29,8 @@ GCC_CFLAGS="-fvisibility=hidden -pipe -Wall -W -Werror \
     -Wstrict-prototypes -Wcast-qual -Wcast-align -Wno-error=unused-function \
     -Wno-error=unused-variable -Wno-error=deprecated-declarations"
 
-LT_INIT
+AC_PROG_LIBTOOL
```

Comment:
Oh, this was by a mistake. I can fix it up.

Linux build needs it because it runs on some special environment to make the shared libraries portable between linux distiributions (more info: https://www.python.org/dev/peps/pep-0513/).

So lines are changed (now the change happens in docker and not affects the working directory directly) before the linux build starts (see dockerfiles in `utils/`).

Reply:
I don't see any mention of libtool in that link? What happens if you try to use LT_INIT instead of AC_PROG_LIBTOOL? I'd have assumed they'd behave identically.

Reply:
So this uses some very old CentOS 5 and LT_INIT simply didn't work there.

Reply:
Ok. But referring to my comment regarding the output of `make dist` - if you generate the source distribution first, and then copy that to the docker environment for building the python packaging, you won't need to run autoreconf, etc. The package will already contain everything needed for libtool, etc, to work and it shouldn't matter if the system is older.

Reply:
Ok, this sounds good.

Reply:
Fixed.